birdhouse/optional-components/scheduler-job-clean_old_files/default.env

```diff
@@ -0,0 +1,24 @@
+export SCHEDULER_JOB_CLEAN_OLD_FILES_IMAGE='${BASH_IMAGE}'
+
+export SCHEDULER_JOB_CLEAN_OLD_FILES_FREQUENCY="5 2 * * 0" # weekly on Sunday at 2:05
+
+export SCHEDULER_JOB_CLEAN_OLD_FILES_VOLUMES='$(for opt in ${SCHEDULER_JOB_CLEAN_OLD_FILES_OPTIONS}; do printf " --volume %s:rw " "$(echo $opt | cut -d\| -f 1)"; done)'
```

Comment:
Oh ! Additional volume mount as well !  I have the same problem with adapting `deploy-data` job.  I am testing my change, once working, I'll send a PR.